### ISSUE TITLE
Overfitting?

### ISSUE THREAD

**Issue by Sheng-cc** [mpc001/end-to-end-lipreading#17]
@mpc001 
Thanks for your code. And I try to replicate your code in keras-tensorflow, but It suffers from serious overfitting. In N2 phase (3DCNN+ResNet+temporal Conv), I fully follow your training settings (data augmentation and ROI cropping), the training accuracy is around 88%, However, the validation accuracy is only about 65%. The result is far from yours (74.6%), I don't know why, so could you tell me the more training detail and tricks?

**Comment by mpc001** [comment]
Hi, Sheng, the training steps and preprocessing steps are all described in that paper. If the training and validation accuracy continuously increase during training, it is still fine in my opinion
